### DISCUSSION TITLE
How to open doors

### DISCUSSION THREAD

**Discussion by methadata** [gicamm/homeassistant-comelit#87] [Q&A] [answered]
According to the [Wiki](https://github.com/gicamm/homeassistant-comelit/wiki), to be able to open a door, we need to create a "Scenario" in SimpleProg first.

I've already created some scenarios in SimpleProg, under the Comelit Hub specifying the ViP address, relay, etc:
![image](https://github.com/gicamm/homeassistant-comelit/assets/3408286/196f1955-ffdd-41ab-9000-b9ecfa7de088)

After configuring it, I wrote the config to the Hub module.

In the wiki, it says that "The comelit integration will automatically discover the new entity 'scene.comelit_opendoor'"

However, the HA integration is not showing anything. I cannot see anything in the Comelit App, nor in the MaxiManager.

Any ideas what I'm missing here to not get the entity in HA?

Thanks in advance!

**Comment by gicamm** [comment]
If you don't see the scenario, either in the application or in the Maxi Manager, the Hub doesn't have the scenario, so you can't see it in the HA.
Has the configuration been written successfully?

**Reply by methadata** [reply]
Thanks for your reply.  You're absolutely right: my issue is not related with the integration, nor HA then.

So now, I don't know what's failing with the process of writing the config to the Comelit Hub from Simpleprog. I've just found that when selecting "Read All Modules" or "Write All modules" buttons, nothing fails, but in the debug window I see the following message between Tx and Rx lines:
```
Rx: Error Connot write data in the transport connection: An error happened trying to connect due to connected party didn't respond properly after some time, or there was an error in the established connection because the connected host wasn't unable to respond
```

However, SimpleProg is able to read and write data from other devices... Have you ever found anything like this?

**Reply by methadata** [reply, marked as answer]
Ok, so it was just that I didn't know how to properly write the config to the Comelit Hub.

Instead of Exporting the config to the Hub using the "Export/Import" button, I was using the "Write all modules" button.

Problem solved, my fault, and I'd like to remove this discussion but I suggest keeping it just in case any other n00b with SimpleProg faces the same issue :)

**Comment by gicamm** [comment]
Thanks for updating the discussion :D
